vendor/github.com/BurntSushi/toml/cmd/toml-test-decoder/COPYING

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Hi @gregwebs , How do you update the vendor with specified package? 
I always follower the steps like:
```
dep ensure -update "github.com/BurntSushi/toml"
dep prune
sh ./hack/clean_vendor.sh
```
but this way will remove the COPYING files.

Reply:
Oh, I didn't know about that. The latest version of the `dep` command tells me that `dep ensure` now automatically does pruning, so it looks like just the clean script needs to be ran.

I would like to send a PR to change the script to not remove legal files. Generally speaking one should not separate code from its license. We cannot really even claim the license is known via Gopkg meta data because the github repo could be deleted.

Reply:
Actually, the removal of COPYING seems like a mistake since we intentionally keep LICENSE. I will send a PR to fix this.

Reply:
https://github.com/pingcap/pd/pull/1213